### PR TITLE
Change main to dev in pr template

### PR DESCRIPTION
**Pull request by YouKnowBlom**
**Changes:**
* Replace main with dev in pr template (Set target branch to dev).

**I have:**

- [x] Set target branch to main.
- [x] Ensured code is formatted, linted.
- [x] Cleaned up git history.
- [x] Ensured commit messages describe *what* was changed, and *why*.
- [x] Ran tests locally, checked output.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 **I have:**
 <!-- fill in with [x] -->
-- [ ] Set target branch to main.
+- [ ] Set target branch to dev.
```
Comment (AxelMatstoms) [inline .github/pull_request_template.md:8]: I would prefer:
"Set target to correct branch, usually `dev`".

**Review by AxelMatstoms — APPROVED** [review]
Looks good, ready to merge.

**Review by fallensach — APPROVED** [review]
looks good!

**Review by TobiasElf — APPROVED** [review]
looks good

**Pull request merged**